eridanus/docker-compose.dev.yml

```diff
@@ -30,10 +30,55 @@ services:
       dataLake:
         aliases:
           - postfix
+  mariadb:
+    container_name: eridanus-mariadb
+    image: mariadb
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: ${ENV_MYSQL_PASSWORD}
+    volumes:
+      - mariadbdata:/var/lib/mysql
+    ports:
+      - 3307:3306
+    networks:
+      local:
```

Comment:
Pretty sure you are going to want this on the dataLake network, not the local network.